plugins/teams/skills/list-regressions/list_regressions.py

```
@@ -587,16 +641,35 @@ def main():
+        return 1
+
+    # If team is specified, look up components for that team
+    components_to_filter = args.components
+    team_name = None
+    if args.team:
+        try:
+            team_name = args.team
+            components_to_filter = get_team_components(args.team)
+            if not components_to_filter:
+                print(f"Error: Team '{args.team}' has no OCPBUGS components mapped.", file=sys.stderr)
+                print(f"The team may not exist or has no components assigned.", file=sys.stderr)
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)